crypto/thread/internal.c

```diff
@@ -101,7 +101,7 @@ int ossl_crypto_thread_clean(void *vhandle)
 
 #else
 
-ossl_inline uint64_t ossl_get_avail_threads(OSSL_LIB_CTX *ctx)
+inline uint64_t ossl_get_avail_threads(OSSL_LIB_CTX *ctx)
```

Comment:
```suggestion
static inline uint64_t ossl_get_avail_threads(OSSL_LIB_CTX *ctx)
```

Reply:
You sure about that?  It's used in `providers/implementations/kdfs/argon2.c`. The one with the leading underscore is internal, this one is lock/callit/unlock

Reply:
No, not sure at all, it was just an odd one out.